src/components/Header/Component.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from "prop-types";
+
+const Wavy = ({ wordLengthFactor }) => {
+  return (
+    <div className="relative">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width={wordLengthFactor}
+        height="21"
+        viewBox="0 0 280 21"
+        fill="none"
+        className="absolute"
+        style={{ bottom: "-18px", left: "-25px" }}
```

Comment:
We should use tailwind css classes instead of `style`.

Reply:
fixed. thank you.